packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/entity-events-to-db.listener.ts

```diff
@@ -29,27 +28,29 @@ export class EntityEventsToDbListener {
 
   @OnEvent('*.created')
   async handleCreate(payload: ObjectRecordCreateEvent<any>) {
-    return this.handle(payload, 'created');
+    return this.handle(payload);
   }
 
   @OnEvent('*.updated')
   async handleUpdate(payload: ObjectRecordUpdateEvent<any>) {
     payload.details.diff = objectRecordChangedValues(
       payload.details.before,
       payload.details.after,
+      payload.objectMetadata,
     );
 
-    return this.handle(payload, 'updated');
+    return this.handle(payload);
   }
 
   // @OnEvent('*.deleted') - TODO: implement when we have soft deleted
   // ....
 
-  private async handle(
-    payload: ObjectRecordCreateEvent<any>,
-    operation: string,
-  ) {
-    if (payload.objectMetadata.isSystem) {
+  private async handle(payload: ObjectRecordCreateEvent<any>) {
+    if (
+      payload.objectMetadata.isSystem &&
+      payload.objectMetadata.nameSingular != 'activity' &&
```

Comment:
Other PR has been opened to introduce isAuditLogged and avoid hardcoded rules like that.

Not isAuditLogged means it'll be forwarded to the timeline building service but doesn't mean it's a linked object that will be copied to its parent timeline